### DISCUSSION TITLE
从1.9.1升级到1.11.1后查询的SQL语法高亮没了

### DISCUSSION THREAD

**Discussion by Ethan30k** [hhyo/Archery#2663] [Q&A] [answered]
<img width="519" alt="Snipaste_2024-05-23_10-51-52" src="https://github.com/hhyo/Archery/assets/24836732/809f84fe-df81-4a5a-a9fa-5e0423ba2dc9">

请问就是这么设计的？还是我哪里配错了？

**Comment by LeoQuote** [comment, marked as answer]
应该是有 bug, 可以看看具体原因, 欢迎 pr

**Comment by Tan-Pchan** [comment]
目前我这边遇到的是macbook的比较新的谷歌浏览器大多都是不高亮且没有提示
